GameServerLib/Logic/GameObjects/Stats/Additive.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LeagueSandbox.GameServer.Logic.GameObjects.Stats
+{
+    public class Additive
```

Comment:
What point does this class have? Right now it's basically just a fancy `float`. Is it to be consistent with multiplicative? If yes, why are some stats Additive and some stats Multiplicative, considering that ideally every stat should support both?

Not really a nitpick, just an interesting design choice.

Reply:
`Multiplicative` is required since:
- Calculations are not as simple as addition and subtraction, 
- I am afraid people writing scripts may not know what stacks additively and what doesn't,
- Trying to remove a 100% multiplicative stack results in `DivisionByZeroException` if we were to use `*=` and `/=`.

Additive is added as a request by @moonshadow565 , saying "why don't you just make an `Additive` class along the `Multiplicative`?" It is not really required and its removal won't make any complexity in the code.